github/commitstatus.go

```diff
@@ -31,6 +32,12 @@ func (e *StatusError) Error() string {
 // API is the GitHub API endpoint.
 const API = "https://api.github.com"
 
+// Maximum number of retries for the retryable http request
+const maxRetries = 3
+
+// Maximum sleep time allowed
+var MaxSleepTime = 15 * time.Minute
```

Comment:
I am exporing here because I used the var in tests which is a pkg with different name (black box testing)